templates/federated_reporting/import.sh

```diff
@@ -52,9 +52,14 @@ declare -a hostkeys
 for file in $dump_files; do
   hostkey=$(basename "$file" | cut -d. -f1)
   hostkeys+=($hostkey)
-  "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb --set "ON_ERROR_STOP=1" \
-                      -c "SELECT ensure_feeder_schema('$hostkey', ARRAY[$table_whitelist]);" \
-    > schema_setup.log 2>&1 || failed=1
+  if [ -z $(psql --csv --tuples-only -U cfapache -d cfdb -c "SELECT hub_id FROM __hubs WHERE hostkey = '$hostkey';") ]; then
+    log "No feeder with hostkey $hostkey found in cfdb.__hubs, skipping the dump file $file, consider deleting this file or re-adding the feeder to superhub"
+    dump_files=$(echo "$dump_files" | sed "s,\s\?$file,," | xargs)
```

Comment:
`alias trim=xargs`, why can't I just have a command called `trim` :p

Reply:
But is it really needed? `$dump_files` has the files on separate lines, right? Why should we change it here?

Reply:
When I was testing I observed the files are space separated so was absolutely required if I were to try remove the first or last files.